docs/reference/NuGet-Client-SDK.md

```diff
@@ -33,12 +33,24 @@ You can find the source code for these packages in the [NuGet/NuGet.Client](http
 > For documentation on the NuGet server protocol, please refer to the [NuGet Server API](~/api/overview.md).
 
 ## Support policy
+The most recent version of NuGet Client SDK is fully supported and can be relied on for bug fixes, updates, and enhancements.
 
-All security bugs should be reported to the Microsoft Security Response Center (MSRC) at [MSRC's report page](https://aka.ms/opensource/security/create-report).
+The recommendation is to use the latest versions of NuGet Client SDK packages, and to examine your project for dependencies on deprecated NuGet Client SDK packages.
+
+Out-of-support NuGet Client SDK packages will be [deprecated on nuget.org](../nuget-org/Deprecate-packages.md#client-experience-for-deprecated-packages).
+
+### Patch Releases
+
+Patched versions of NuGet Client SDK will be released exclusively when critical bugs or security fixes are required for a long-term support (LTS) version of Visual Studio or .NET SDK.
+
+All security bugs should be reported to the Microsoft Security Response Center (MSRC) at [https://aka.ms/opensource/security/create-report].
 Also, see the [security policy in the NuGet.Client repo](https://github.com/NuGet/NuGet.Client/blob/dev/SECURITY.md).
 
-We do not guarantee API stability, as our team's responsibility is tooling, not libraries.
-See the [NuGet SDK documentation in the NuGet.Client repo](https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md) for more information.
+### Package Deprecation
+
+As of January 31, 2024, older versions of NuGet Client SDK packages that are not tied to an LTS version of either Visual Studio or .NET have been deprecated.
```

Comment:
Unfortunately, this work was partially done. We can certainly complete package deprecation by end of May 2024 at the latest.

Reply:
Hmm...maybe we remove this? 
Since this would become a JDI.

Reply:
Removed the date entirely